src/components/YieldPools/FarmingPoolAPRCell.tsx

```diff
@@ -139,4 +144,30 @@ const FarmingPoolAPRCell: React.FC<Props> = ({ poolAPR, fairlaunchAddress, pid,
   )
 }
 
+export const ClassicFarmingPoolAPRCell = ({ poolAPR, farm }: { poolAPR: number; farm: Farm }) => {
+  const theme = useTheme()
+  const lpTokenRatio = new Fraction(
+    farm.totalStake.toString(),
+    JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(18)),
+  ).divide(
+    new Fraction(parseUnits(farm.totalSupply, 18).toString(), JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(18))),
+  )
+  const liquidity = parseFloat(lpTokenRatio.toSignificant(6)) * parseFloat(farm.reserveUSD)
+  const farmAPR = useFarmApr(farm, liquidity.toString())
+
+  return (
+    <Flex
+      alignItems={'center'}
+      sx={{
+        gap: '4px',
+      }}
+    >
+      <Text as="span">{(poolAPR + farmAPR).toFixed(2)}%</Text>
```

Comment:
What is this? "+" look weird.

Reply:
that's total apr
farmApr + poolAPR

Reply:
<img width="521" alt="image" src="https://user-images.githubusercontent.com/11427213/208584170-9c22e18f-dbe0-4180-805e-0bc323036781.png">